addons/knowledge/controllers/main.py

```diff
@@ -61,6 +61,24 @@ def redirect_to_article(self, article_id):
             'show_sidebar': bool(self._get_root_articles(limit=1))
         })
 
+    @http.route('/knowledge/article', type='http', auth='user')
```

Comment:
Would name it /home (/article denotes a specific article)

Reply:
yeah, I was not suuure heeeee